src/python/WMCore/WMRuntime/Timestamps.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+_Timestamps_
+
+"""
+
+import getopt
+import logging
+import pickle
+import sys
+
+
+# script options
+options = {
+        "reportIn=": "",      # FJR input file
+        "reportOut=": "",     # FJR output file
+        "timeStartSec=": "",  # UNIX seconds of the start
+        "timeEndSec=": "",    # UNIX seconds of the end
+        }
+
+
+def addTimestampMetrics(config, timeStartSec, timeEndSec):
+    """
+    Adjust timestamp metrics of provied FJR data.
+
+    :param config: input FJR data
+    :param timeStartSec: start timestamp in UNIX seconds
+    :param timeEndSec: end timestamp in UNIX seconds
+    """
+    timing = config.section_('Timing')
```

Comment:
It might be useful to rename this section to something like `WMTiming`, such that we can eventually distinguish between WM and CMSSW-based timing metrics.
I am not sure, but maybe we should do the same for `TotalWallClockTime` and rename it to `WMTotalWallClockTime`, in case it at some point goes out of WMTiming context.

Reply:
done